src/shared/telemetry/generate_types.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+CWD=`pwd`
+BASEDIR=`dirname $0`
+JSON_FILE=$BASEDIR/service-2.json
+DATE=$(jq -r '.metadata.apiVersion' $JSON_FILE)
+SDK_MODULE_NAME=aws-sdk
+SERVICE='ClientTelemetry'
+SERVICE_LOWERCASE=`echo $SERVICE | tr '[:upper:]' '[:lower:]'`
+AWS_SDK_DIR=aws-sdk-js
+TYPEFILE=$BASEDIR/$SERVICE_LOWERCASE.d.ts
+
+# get a fresh sdk
+git clone --depth 1 https://github.com/aws/aws-sdk-js.git
```

Comment:
since you `cd` and `rm -rf` against `$AWS_SDK_DIR`, it might make sense to explicitly clone into that location as well

Reply:
Sure, we can do that.